kqueen/engines/openstack_kubespray.py

```diff
@@ -80,7 +80,6 @@ class OpenstackKubesprayEngine(BaseEngine):
                 "help_message": "Must be odd number",
                 "validators": {
                     "required": True,
-                    "min": 3,
                     "parity": "odd",
```

Comment:
why min is not needed?

Reply:
Because kubespray can deploy cluster with 1 master node

Reply:
So please remove or from validation on server-side https://github.com/redixin/kqueen/blob/58ee6824412c298919ef5db5d893e88a8d7256ab/kqueen/engines/openstack_kubespray.py#L599